src/sagemaker/workflow/pipeline.py

```diff
@@ -824,15 +828,34 @@ def list_steps(self):
         """Describes a pipeline execution's steps.
 
         Returns:
-             Information about the steps of the pipeline execution. See
+             Information about the steps of the pipeline execution in a Pandas table. See
              `boto3 client list_pipeline_execution_steps
```

Comment:
`in a table.`

Usually we don't put details like which library/tool in the doc string in case we change it in the future.